roles/openshift_logging/tasks/set_defaults_from_current.yml

```diff
@@ -176,13 +176,6 @@
     - set_fact:
         __openshift_logging_eventrouter_nodeselector: "{{ openshift_logging_facts['eventrouter']['deploymentconfigs']['logging-eventrouter']['nodeSelector'] | default('') | from_yaml }}"
 
-    - conditional_set_fact:
```

Comment:
why is this being removed as part of fixing #9224 ?

Reply:
IIUC this would use existing settings from the eventrouter for CPU and memory limits. 
Do we still want this, although ansible vars are set a different value?
Did I misunderstood that part?

Reply:
Part of what we try to do with a logging installation/upgrade is to maintain any tuning that a customer may have done as part of their installation but may not have moved into an ansible variable. So we will use the existing values as a default value instead of the default value provided by the role (if installed), however we will always use the inventory value over any defaults.